config/default_records/tube_purposes/004_illumina_htp_legacy_purposes.dev.yml

```diff
@@ -28,7 +28,7 @@ Lib Pool SS-XP:
   target_type: StockMultiplexedLibraryTube
   stock_plate: false
 Cap Lib Pool Norm:
-  type: IlluminaHtp::MxTubeNoQcPurpose
+  type: IlluminaHtp::MxTubePurpose
   target_type: MultiplexedLibraryTube
```

Comment:
MxTubeNoQcPurpose behaviour was migrated back to what was previously the parent class MxTubePurpose. Decided to go for the more straight forward name